.github/workflows/checkDocSync.yml

```diff
@@ -15,9 +15,6 @@
 name: Check Documentation Synchronized
 on:
   pull_request:
-    branches:
-      - v0
-      - main
 jobs:
```

Comment:
I am assuming with this change, these will only get triggered for `main` branch now ?

Reply:
Please describe the strategy across workflows: https://github.com/GoogleContainerTools/kpt/issues/2229

Reply:
Updated PR description. The changes in this PR don't affect workflows in `v0`, but future branches of `main` (expected to fall under either development or legacy branches) will pick up the same behavior and run when they're modified.